packages/components/src/Link/_docs/Link.stickersheet.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { type Meta } from '@storybook/react'
+import { within } from '@storybook/test'
+import { Icon } from '~components/__rc__/Icon'
+import { StickerSheet, type StickerSheetStory } from '~storybook/components/StickerSheet'
+import { Link, type LinkProps } from '../Link'
+
+export default {
+  title: 'Components/Link',
+  component: Link,
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const variants = ['primary', 'secondary'] satisfies LinkProps['variant'][]
+const sizes = ['extra-small', 'small', 'body', 'intro-lede'] satisfies LinkProps['size'][]
+const href = 'https://www.google.com' satisfies LinkProps['href']
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <>
+      <StickerSheet
+        title="Link"
+        headers={['Icon start', 'Icon end', 'isUnderlined', 'isDisabled']}
+        isReversed={isReversed}
+      >
+        {variants.map((variant) =>
+          sizes.map((size) => (
+            <StickerSheet.Row key={size + variant} header={`${variant} (${size})`}>
+              <Link
+                variant={variant}
+                size={size}
+                href={href}
+                isUnderlined={false}
+                isReversed={isReversed ? isReversed : false}
+                isInline={false}
+                icon={<Icon name="add" isPresentational />}
+                iconPosition={'start'}
```

Comment:
Don't need the curly brackets if it's just a string can just do `iconPosition="start"` same for the rest of the examples below.